docs/developer-docs/emulator-setup-for-developers.md

```diff
@@ -75,22 +75,30 @@ When an update for RAIntegration is available, it will ask you if you want to up
 
 ## PPSSPP
 
-PPSSPP currently requires a development build for RAIntegration support:
+PPSSPP supports RAIntegration in its main builds now as of version 1.18. You can grab the latest version here:
 
-- **https://www.ppsspp.org/devbuilds/**
-
-Ensure that the file is named `RA_Integration-x64.dll` and place it in PPSSPP's main folder.
-
-![RAIntegration in PPSSPP folder](/public/ppsspp-raintegration-folder.png)
+- **https://www.ppsspp.org/download/**
 
-Then launch `PPSSPPWindows64.exe` and go to Settings -> Tools -> RetroAchievements. Here you can log in under the Account tab. Under the Developer Tools tab, you can enable RAIntegration. Enabling unofficial achievements may also be necessary in order to view them with the developer tools.
+After running the installer, launch PPSSPPP and go to Settings -> Tools -> RetroAchievements. Here you can log in under the Account tab. Under the Developer Tools tab, you can enable RAIntegration. Enabling unofficial achievements may also be necessary in order to view them with the developer tools.
 
 ![RAIntegration setting in PPSSPP](/public/ppsspp-raintegration-setting.png)
 
 Afterwards, exit PPSSPP and reopen it to load RAIntegration. RetroAchievements will have its own menu with the developer tools.
 
 ![RAIntegration menu in PPSSPP](/public/ppsspp-raintegration-menu.png)
 
+### Portable Install
+
+The installer from here downloads PPSSPP as an installer. If you want to avoid installing PPSSPP and run it as a portable application in its own folder, you can grab one of the development builds from here:
```

Comment:
This sentence reads kinda weird. Wondering if we can tighten this up a bit. Something to the effect of:

> To download PPSSPP as a portable application, bypassing the need for installation, you can grab one of the development builds from here. Otherwise, you can download the installer version from this link:

Reply:
I think I can do slightly better than that and pull this into the section above, on a reread I'm not sure this needs to be its own section. Thinking from a loose structure:

1. Tell the reader where to get the emulator
2. Tell the reader how to set up the emulator via the emulator itself
3. If the reader chose a manual install, give the reader instructions on how to ensure the manual install functions.